package.json

```diff
@@ -24,6 +24,7 @@
   "homepage": "https://github.com/dabit3/React-Native-Elements#readme",
   "dependencies": {
     "react-native-side-menu": "^0.20.1",
-    "react-native-tab-navigator": "^0.3.3"
+    "react-native-tab-navigator": "^0.3.3",
+    "react-native-vector-icons": "^4.0.0"
```

Comment:
@jenessawhite *react-native-vector-icons* is a peer dependency. No need to add it as a dependency